rwkv.cpp

```diff
@@ -615,25 +646,106 @@ void rwkv_ctx_size_add_tensor(struct rwkv_ctx_size & size, const uint64_t tensor
     rwkv_ctx_size_add_tensor(size, tensors, views, rwkv_type_to_ggml[header.data_type], header.width, header.height);
 }
 
-struct rwkv_ctx_size rwkv_single_att_size(const size_t n_embed = 0) {
-    size_t ptr_nelem = sizeof(void *) / sizeof(uint32_t);
-
+struct rwkv_ctx_size rwkv_xx_size(const size_t n_embed = 0, const size_t sequence_len = 1) {
```

Comment:
`rwkv_xx` is confusing to me, but I don't know how to call it better. It's okay to leave it as is

Reply:
it outputs a variable called xx but it could be called something like `input_step`